code/game/machinery/OpTable.dm

```diff
@@ -31,8 +31,9 @@
 	patient = null
 	return ..()
 
-/obj/machinery/optable/detailed_examine()
-	return "Click your target and drag them onto the table to place them onto it."
+/obj/machinery/optable/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>Click-drag someone to the table to place them on top of the table.</span>"
```

Comment:
```suggestion
	. += "<span class='notice'><b>Click-drag</b> someone to the table to place them on top of the table.</span>"
```

Most commands (Click-drag) are boldened in the game already, let's keep up this I think, because it makes it more readable as well. (The player only needs to take a glance at the text to see what they need to do).

Reply:
reviews should be adressed